temporal/t.rast.series/t.rast.series.html

```diff
@@ -11,6 +11,18 @@ <h2>DESCRIPTION</h2>
 <em>t.rast.series</em> is a simple wrapper for the raster module
 <b>r.series</b>. It supports a subset of the aggregation methods of
 <b>r.series</b>.
+  
+<h2>NOTES</h2>
+
+To avoid problems with too many open files, by default, the maximum 
+number of open files is set to 1000. If the number of input raster 
+files exceeds this number, the z-flag will be invoked. Because this 
+will slow down processing, the user can set a higher limit with the 
+'open files' option. Note that 'open file' limit should not exceed the 
```

Comment:
```suggestion
files exceeds this number, the <b>-z</b> flag will be invoked. Because this 
will slow down processing, the user can set a higher limit with the 
<b>open files</b> parameter. Note that <b>open file</b> limit should not exceed the 
```

Please follow the https://trac.osgeo.org/grass/wiki/Submitting/Docs#Markupstyleguide